examples/applications/plot_face_recognition.py

```diff
@@ -46,7 +46,7 @@
 logging.basicConfig(level=logging.INFO, format='%(asctime)s %(message)s')
 
 
-# #############################################################################
+# %%
```

Comment:
@rth , as noted by @thomasjpfan and @lucyleeow these ones should actually be interpreted as plain comments (this is on purpose according to https://github.com/scikit-learn/scikit-learn/pull/9061)

I can try to revert if you want

Reply:
They should be all in the first commit dd7cb01 (as I made a mistake in the regex the first time). If they are plain comments why do we need 80 chars of "#"?  For instance in https://scikit-learn.org/dev/auto_examples/cluster/plot_mini_batch_kmeans.html I would just remove them, saying that we `# Compute clustering with MiniBatchKMeans` doesn't actually need all those `#`. We don't really comment that way usually, so I'm not sure why put them in examples. Also it's confusing because I really though they were there for sphinx. 

But anyway it's a separate discussion happy to revert in this PR.